src/luv.h

```diff
@@ -54,21 +54,28 @@
 #pragma clang diagnostic push
 #pragma clang diagnostic ignored "-Wunused-function"
 #endif
+
 /* There is a 1-1 relation between a lua_State and a uv_loop_t
    These helpers will give you one if you have the other
    These are exposed for extensions built with luv
    This allows luv to be used in multithreaded applications.
 */
 LUALIB_API lua_State* luv_state(uv_loop_t* loop);
-/* All libuv callbacks will lua_call directly from this root-per-thread state
-*/
+
+/* All libuv callbacks will lua_call directly from this root-per-thread state */
 LUALIB_API uv_loop_t* luv_loop(lua_State* L);
 
+/* Set an extern uv_loop_t in a lua_State
+   This must be called before luaopen_luv, luaopen_luv don't init an inner loop
```

Comment:
maybe something like
```
This must be called before luaopen_luv, so luaopen_luv doesn't init an own loop
```

Reply:
ok, my language barrier

Reply:
np, I'm not native speaker too, was just a minor suggestion